src/constants/metadata.ts

```diff
@@ -6,73 +6,237 @@ export const NETWORK_METADATA: Record<string, NetworkMetadata> = {
     [NETWORKS.ETHEREUM.MAINNET]: {
         name: 'Ethereum Mainnet',
         currency: 'ETH',
-        icon: 'https://assets.coingecko.com/coins/images/279/small/ethereum.png',
-        explorerUrl: 'https://etherscan.io'
+        icon: 'https://ethereum.org/static/6b935ac0e6194247347855dc3d328e83/13c43/eth-diamond-black.png',
+        explorerUrl: 'https://etherscan.io',
+        rpcUrl: 'https://ethereum.publicnode.com'
     },
     [NETWORKS.BSC.MAINNET]: {
         name: 'BNB Smart Chain',
         currency: 'BNB',
         icon: 'https://assets.coingecko.com/coins/images/825/small/bnb-icon2_2x.png',
-        explorerUrl: 'https://bscscan.com'
+        explorerUrl: 'https://bscscan.com',
+        rpcUrl: 'https://bsc-dataseed.binance.org'
     },
     [NETWORKS.POLYGON.MAINNET]: {
         name: 'Polygon',
         currency: 'MATIC',
-        icon: 'https://assets.coingecko.com/coins/images/4713/small/matic-token-icon.png',
-        explorerUrl: 'https://polygonscan.com'
+        icon: 'https://polygon.technology/favicon.png',
+        explorerUrl: 'https://polygonscan.com',
+        rpcUrl: 'https://polygon-rpc.com'
+    },
+    [NETWORKS.AVALANCHE.MAINNET]: {
+        name: 'Avalanche C-Chain',
+        currency: 'AVAX',
```

Comment:
@dio-dev what do you think if in metadata there will be also native currency decimals?